client/landing/gutenboarding/constants.ts

```diff
@@ -13,7 +13,7 @@ const fontTitles: Partial< Record< Font, string > > = {
 	'Playfair Display': 'Playfair',
 };
 
-export const PAID_DOMAINS_TO_SHOW = 5;
+export const domainIsAvailableStatus = [ 'available', 'available_premium' ];
```

Comment:
I have only brought in the available status' rather than all of the status' which can be found [here](https://github.com/Automattic/wp-calypso/blob/f2c9d99b07e14ea938829e9b2e5b31b6fb388a90/client/lib/domains/constants.js) as we are only concerned if they're available.

See existing check here https://github.com/Automattic/wp-calypso/blob/f2c9d99b07e14ea938829e9b2e5b31b6fb388a90/client/components/domains/register-domain-step/index.jsx#L759

Is this enough? And do we want to put `unknown` into this list as well. `unknown` gets returned in the `catch {}` of the `isAvailable()` request [here](https://github.com/Automattic/wp-calypso/pull/41548/files#diff-d053ea2b31eaea5984a8f2c514e52a1cR27)

Reply:
IMO I wouldn't add `unknown` as part of `domainIsAvailableStatus`, better safe than sorry in this case. 

We could think about how to deal with other statuses in another PR. This is good for now. :)

Reply:
Not a blocker, but perhaps `domainIsAvailableStatus` could be imported from `packages/domain-picker/src/constants.ts` since these 2 lines are duplicated? 

Though I also understand that original source is coming from `lib/domains/constants`, and we can't import that in the package so there's gonna be duplication anyhow.